weareinreach/inreach-catalog
# Removed serviceAreaCoverage.json

## Description
This removes the serviceAreaCoverage.json file and the corresponding checks in api.js. It looks like this was only used for validation, so removing it should be backward compatible. If we accidentally end up checking a service area that's not present on any of the organizations in the database, we should just get no results. 

- Asana ticket:
https://app.asana.com/0/1132189118126148/1180818950425345

## PR Checklist


## How to Test
I have tested that search still works in the catalog app, but I'm not sure if I'm hitting the particular edge case that this serviceAreaCoverage check was meant to address, so I might need some help with a test plan.
